plat/xen/include/xen/intctrl.h

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
This seems like a duplicate of [this source file](https://github.com/unikraft/unikraft/blob/staging/plat/kvm/include/kvm/intctrl.h). Maybe @michpappas has a stronger view if we should duplicate or move the file on an upper level

Reply:
Hm. It is. But moving this to the upper level will require reworking kvm code as well. that's why I've added my implementation. Let's ask @michpappas how it can be handled.

Reply:
Let's leave it as is for `0.14`, we'll be introducing `ukintctlr` in `0.15` (hopefully) and we can consider that then.

Reply:
ok